### PR TITLE
Add Ethan as an author; article about randomness in tests.

### PR DESCRIPTION
**Pull request by genericmoniker**
Adds Ethan as an author in anticipation of his SaintCon article. 
Adds an article about randomness in testing.

### CODE REVIEW

File: pelicanconf.py [hunk 1]
```diff
@@ -46,15 +46,31 @@
...
+    },
+    'Ethan': {
+        'description': """
+            Ethan Smith is a computer science student at Southern Utah 
+            University and vice president of the school's Cybersecurity Club.
+        """,
+        'cover': '/site/esmithy.jpg',
+        'image': '/site/ethan-small.png',
+        'links': (
+            ('github', 'https://github.com/genericmoniker'),
```
Comment (ethjsmith) [inline pelicanconf.py]: ('github', 'https://github.com/urd000med/'),

**Review by ethjsmith — DISMISSED** [review]
looks good, other than my github link is wrong... is adding a new article as easy as making a new text document in the content folder, IE `content/greatarticle.md`?

File: pelicanconf.py (hunk 1 above)
Comment (genericmoniker) [reply pelicanconf.py]: OK, fixed the github link in the latest commit.

**Comment by genericmoniker** [comment]
@urd000med 

For adding an article -- yes, that's the general idea: create a new Markdown file. It needs to have the "header" information like the other articles. 

I haven't yet added an automated build that will deploy upon merging to master, so I have to do that locally for now.

**Pull request merged**